src/providers/local/staking/userUserStakingData.ts

```diff
@@ -22,6 +22,7 @@ import { getGaugeAddress } from './staking.provider';
 export type UserGuageShare = {
```

Comment:
```suggestion
export type UserGaugeShare = {
```

Sidenote but I just noticed we have a typo here